credits.txt

```diff
@@ -11,6 +11,8 @@ Photos from Wikimedia Commons
   Matthew Field (CC-BY-SA 3.0)
 and other photographers
   Chris Pickel (CC-BY-SA 4.0)
+  Michael Wilson (CC-BY-4.0)
+  @Gardevou (CC-BY-4.0)
```

Comment:
isn't it Gardevour? (also in the fields17.jpg & copyright) it is on Discord so that's why